AsyncDisplayKit/Private/ASControlTargetAction.m

```diff
@@ -0,0 +1,66 @@
+//
+//  ASControlTargetAction.m
+//  AsyncDisplayKit
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#import "ASControlTargetAction.h"
+
+@implementation ASControlTargetAction
+{
+  id _target;
+  BOOL _createdWithNoTarget;
+}
+
+- (void)setTarget:(id)target {
```

Comment:
@tamaskarai methods in this file should match the style of having method open braces on the next line.  Control statements have them on the same line though.